dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -228,67 +219,66 @@ private BulkActionView buildBulkActionView (final SearchResponse response,
 
             try {
                 final WorkflowStep workflowStep = this.workflowAPI.findStep(stepCount.getKey());
-                if( archivedSchemes.contains(workflowStep.getSchemeId())){
-                    Logger.info(getClass(),()-> "Step with id "+ stepCount.getKey() + " is linked with an Archived WF and will be skipped." );
+                if (archivedSchemes.contains(workflowStep.getSchemeId())) {
+                    Logger.info(getClass(), () -> "Step with id " + stepCount.getKey()
+                                    + " is linked with an Archived WF and will be skipped.");
                     continue;
                 }
 
-                final CountWorkflowStep step = new CountWorkflowStep(stepCount.getValue(),
-                        workflowStep);
-                stepActionsMap.put(step, this.workflowAPI.findBulkActions(step.getWorkflowStep(), user)
-                        .stream().filter(WorkflowAction::shouldShowOnListing)
-                        .collect(CollectionsUtils.toImmutableList()));
-            }catch (Exception e){
-                Logger.warn(getClass(),()-> "Unable to load step with id "+ stepCount.getKey() + " The index is slightly out of sync." );
+                final CountWorkflowStep step = new CountWorkflowStep(stepCount.getValue(), workflowStep);
+                stepActionsMap.put(step, this.workflowAPI.findBulkActions(step.getWorkflowStep(), user).stream()
+                                .filter(WorkflowAction::shouldShowOnListing).collect(CollectionsUtils.toImmutableList()));
+            } catch (Exception e) {
+                Logger.warn(getClass(), () -> "Unable to load step with id " + stepCount.getKey()
+                                + " The index is slightly out of sync.");
             }
         }
 
-        return (stepCounts.size() > 0 ?
-            this.buildBulkActionView(stepActionsMap) :
-            new BulkActionView(Collections.emptyMap())
-        );
+        return (stepCounts.size() > 0 ? this.buildBulkActionView(stepActionsMap) : new BulkActionView(Collections.emptyMap()));
```

Comment:
Codacy found an issue: [Substitute calls to size() == 0 (or size() != 0, size() > 0, size() < 1) with calls to isEmpty()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)